libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/model/Config.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.navigation.route.onboard.model
+
+import com.mapbox.navigation.navigator.model.RouterConfig
+
+data class Config(
```

Comment:
This data class and the `RouterConfig` data class are unnecessary. We can just use `RouterParams` that come with the navigator dependency.

Reply:
actually we cannot in this version, because we don't provide to user opportunity to replace our HttpClient

Reply:
and that's fine, as long as it's all contained in the `onboard` module